src/common/Tile/index.js

```diff
@@ -157,7 +157,7 @@ export const DetailsTile = (
 						<TileDataContent>
 							<TileDataTitle>
 								{
-									movieTile ? `Production: ` : `Date of birth: `
+									movieTile ? `Production: ` : `Place of birth: `
 								}
```

Comment:
Thanks for correcting that mistake, actually this line should be:
```
movieTile ? `Release date: ` : `Place of birth: `
```

Reply:
You're welcome! 🙌 I appreciate your feedback, and I've duly noted the correction. I'll make sure to update the line to:
`movieTile ? `Release date: ` : `Place of birth: ``
Thanks for catching that! 😊🚀